doc/ref/start-of-release.md

```diff
@@ -0,0 +1,43 @@
+% Explanation of start of release changes
+
+## Overview
+
+The start of release changes, the changes that turn JDK N into JDK
+(N+1), are primarily small updates to various files along with new files to
+store symbol information to allow `javac --release N ...` to run on
+JDK (N+1).
+
+As a matter of policy, there are a number of semantically distinct
+concepts which separately get incremented at the start of a new release:
+
+* Feature value of `Runtime.version()`
+* Highest class file format major version recognized by the platform
+* Highest `-source`/`-target'/`--release` argument recognized by `javac`
+
+In more detail, updated files include:
```

Comment:
The phrasing here, "... updated files include", suggests that this will just be a list of files, but in fact it's a list of files and instructions for updating them.